x/commitment/client/cli/tx_update_vesting_info.go

```diff
@@ -0,0 +1,137 @@
+package cli
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/elys-network/elys/x/commitment/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdUpdateVestingInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-vesting-info",
+		Short: "Broadcast message update-vesting-info",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			title, err := cmd.Flags().GetString(cli.FlagTitle)
+			if err != nil {
+				return err
+			}
+
+			summary, err := cmd.Flags().GetString(cli.FlagSummary)
+			if err != nil {
+				return err
+			}
+
+			metadata, err := cmd.Flags().GetString(cli.FlagMetadata)
+			if err != nil {
+				return err
+			}
+
+			argBaseDenom, err := cmd.Flags().GetString("base-denom")
+			if err != nil {
+				return err
+			}
+
+			argVestingDenom, err := cmd.Flags().GetString("vesting-denom")
+			if err != nil {
+				return err
+			}
+
+			argEpochIdentifier, err := cmd.Flags().GetString("epoch-identifier")
+			if err != nil {
+				return err
+			}
+
+			argNumEpochs, err := cmd.Flags().GetString("num-epochs")
+			if err != nil {
+				return err
+			}
+
+			argVestNowFactor, err := cmd.Flags().GetString("vest-now-factor")
+			if err != nil {
+				return err
+			}
+
+			argNumMaxVestings, err := cmd.Flags().GetString("num-max-vestings")
+			if err != nil {
+				return err
+			}
+
+			signer := clientCtx.GetFromAddress()
+			if signer == nil {
+				return errors.New("signer address is missing")
+			}
+
+			govAddress := sdk.AccAddress(address.Module("gov"))
```

Comment:
Would be good to use govtypes.ModuleName instead of "gov"

Reply:
Done